services/datamanager/datasync/upload_arbitrary_file.go

```diff
@@ -5,14 +5,21 @@ import (
 	"io"
 	"os"
 	"path/filepath"
+	"time"
 
+	clk "github.com/benbjohnson/clock"
 	"github.com/pkg/errors"
 	v1 "go.viam.com/api/app/datasync/v1"
 )
 
 // UploadChunkSize defines the size of the data included in each message of a FileUpload stream.
 var UploadChunkSize = 64 * 1024
 
+// Default time to wait in milliseconds to check if a file has been modified.
+const defaultFileLastModifiedMillis = 10000.0
```

Comment:
I know I set this originally, but I'm wondering if this default should be lower. 10 seconds seems like a long time in the context of users may be moving files to the directory they wish to sync content from and then receive a bunch of errors. Maybe a product question?

Reply:
Good question, but I would prefer to err on the side of being conservative for the use case of files that are being appended to. If someone moves their file, they'll receive errors which'll make sense (since they just moved them) but then it'll resync again later.